.github/workflows/backwards_compatibility_tests_workflow.yml

```diff
@@ -15,8 +15,8 @@ jobs:
       matrix:
         java: [ 21, 23 ]
         os: [ubuntu-latest]
-        bwc_version : ["2.9.0","2.10.0","2.11.0","2.12.0","2.13.0","2.14.0","2.15.0","2.16.0","2.17.0","2.18.0","2.19.0","2.20.0-SNAPSHOT"]
-        opensearch_version : [ "3.0.0-SNAPSHOT" ]
+        bwc_version : [ "2.9.0","2.10.0","2.11.0","2.12.0","2.13.0","2.14.0","2.15.0","2.16.0","2.17.0","2.18.0","2.19.0","2.20.0-SNAPSHOT" ]
```

Comment:
we should add 3.0.0 to bwc_version list, 3.1 has to be backward compatible with 3.0